space_packet_parser/common.py

```diff
@@ -42,6 +42,7 @@ def from_xml(
             tree: Optional[ElementTree.ElementTree],
             parameter_lookup: Optional[dict[str, any]],
             parameter_type_lookup: Optional[dict[str, any]],
+            container_lookup: Optional[dict[str, any]],
```

Comment:
Does `any` actually work with mypy, I thought it wanted `typing.Any`.

A few minor nits/questions:
* Do we want to call these "parameters", "parameter_types", "containers" so it is equivalent to what is stored on the definition? (Thinking of this now before we make a major version release)
* Should these be typed as `dict[str, Parameter]`, `dict[str, ParameterType]`, `dict[str, Parameter | SequenceContainer]`
* Since many of these are repeated all over the place, do you want to bring any of these types into a typing helpers / commons import area so you don't have to redefine them all the time?

Reply:
I believe `any` is valid the same way `dict` and `list` are valid. There's also a `typing.Any` similar to `typing.Dict` and `typing.List`.

Let's address this in a future PR.

Reply:
...Before we release a new major version.

Reply:
`any` is a function, not a type though. I've had mypy complain in the past for any (maybe we need a mypy type-checking action too), but perhaps they've changed the typing landscape now.